Gopkg.toml

```diff
@@ -65,9 +65,8 @@
 
 [[constraint]]
   name = "github.com/deislabs/cnab-go"
-  source = "github.com/radu-matei/cnab-go"
-  branch = "add-duffle-types"
-
+  revision = "9028f0f800a40977c2e49a38be764978cfe82324"
```

Comment:
I'd much prefer to see us tag a version number on cnab-go and then change this to `version = "0.1.0"`.

Reply:
I tried to avoid adding a version, since it would introduce yet another one that is not the same with the spec or Duffle...

Reply:
Ok, what tag version number do you suggest for now?

The only thing I don't really know how to avoid is confusion from introducing yet another version number, not in sync with the spec or Duffle..

Reply:
Pinning to a specific version causes problems with dep's solving algorithm which is why I am so strongly saying don't do that.

When in doubt, use the semver, Radu. If your plan was to use the exact same version # between duffle, cnab-spec and cnab-go (not sure I agree) then stick with what we are doing with the other tools and version it with `0.1.0-ralpha.1+englishrose`. Or if you want to be forward looking, `1.0.0-alpha1` and just keep bumping that last # until the cnab spec is released at v1, then after that you can use the minor and patch since cnab doesn't use those values and only has a major.

Other strategies:

* `0.1.0+cnab1` lets you add build metadata indicating compatibility with cnab v1 without triggering prerelease handling.
* Dual tagging like k8s client does, where you have a version number that makes sense for your own client, and then non semver tags to indicate compatibility with cnab spec versions.